src/points/selectors.ts

```diff
@@ -1,5 +1,44 @@
+import { createSelector } from 'reselect'
+import { PointsMetadata, isPointsActivity } from 'src/points/types'
 import { RootState } from 'src/redux/reducers'
 
 export const nextPageUrlSelector = (state: RootState) => {
   return state.points.nextPageUrl
 }
+
+const pointsConfigSelector = (state: RootState) => state.points.pointsConfig
+
+export const pointsSectionsSelector = createSelector(
+  [pointsConfigSelector],
+  (pointsConfig): PointsMetadata[] => {
+    const pointsMetadata: PointsMetadata[] = []
+    if (!pointsConfig) {
+      return pointsMetadata
```

Comment:
Could we change the type of the `pointsConfig` state from `pointsConfig: PointsConfig | null` to `pointsConfig: PointsConfig` and set the initial state to `{ activitiesById: {}}` to avoid this special check? Not sure if it would make things uglier in the long run/not committed to the idea, just curious.

Reply:
you're right, i can't think of any benefits to having an explicit null